overlord/snapstate/snapmgr.go

```diff
@@ -680,10 +680,10 @@ func Manager(st *state.State, runner *state.TaskRunner) (*SnapManager, error) {
 	// FIXME: drop the task entirely after a while
 	runner.AddHandler("clear-aliases", func(*state.Task, *tomb.Tomb) error { return nil }, nil)
 	runner.AddHandler("set-auto-aliases", m.doSetAutoAliases, m.undoRefreshAliases)
-	runner.AddHandler("setup-aliases", m.doSetupAliases, m.doRemoveAliases)
+	runner.AddHandler("setup-aliases", m.doSetupAliases, m.undoSetupAliases)
```

Comment:
looking further I'm not sure we fully need this change because setup-aliases is either used alone, in which case it's undo is removeAliases or is combined with set-auto-aliases which seems to have an asymmetric behavior, in the sense that the path forward doesn't touch the disk but it's undo is undoRefreshAliases that seems to do things with aliases?

Reply:
`undoRefreshAliases` behavior depends on it's context, when used as undo for `set-auto-aliases` it doesn't touch disk, only reverts updated snap state aliases and the `AutoAliasesDisabled` flag.

https://github.com/snapcore/snapd/blob/0238dca2a712505ae0511f14ea5beef7d82c09ca/overlord/snapstate/handlers.go#L3628-L3636